common/views/components/RelevanceRater/RelevanceRater.js

```diff
@@ -45,99 +26,27 @@ const RelevanceRater = ({
   workType,
   _queryType,
 }: Props) => {
+  const [showRatings, setShowRatings] = useState(false);
+  useEffect(() => {
+    setShowRatings(true);
+  }, []);
   return (
-    <div>
-      <RelevanceRaterStyle>
-        <RelevanceRating
-          v={{
-            size: 's',
-            properties: ['padding-top', 'padding-bottom'],
-          }}
-          h={{ size: 'm', properties: ['padding-left', 'padding-right'] }}
-          as="button"
-          index={0}
-          onClick={() =>
-            trackRelevanceRating({
-              id,
-              position,
-              rating: 1,
-              query,
-              page,
-              workType,
-              _queryType,
-            })
-          }
-        >
-          No apparent relationship to search term
-        </RelevanceRating>
-        <RelevanceRating
-          v={{
-            size: 's',
-            properties: ['padding-top', 'padding-bottom'],
-          }}
-          h={{ size: 'm', properties: ['padding-left', 'padding-right'] }}
-          as="button"
-          index={1}
-          onClick={() =>
-            trackRelevanceRating({
-              id,
-              position,
-              rating: 2,
-              query,
-              page,
-              workType,
-              _queryType,
-            })
-          }
-        >
-          Reasonable to be retrieved but should not be this highly ranked
-        </RelevanceRating>
-        <RelevanceRating
-          v={{
-            size: 's',
-            properties: ['padding-top', 'padding-bottom'],
-          }}
-          h={{ size: 'm', properties: ['padding-left', 'padding-right'] }}
-          as="button"
-          index={2}
-          onClick={() =>
-            trackRelevanceRating({
-              id,
-              position,
-              rating: 3,
-              query,
-              page,
-              workType,
-              _queryType,
-            })
-          }
-        >
-          Not perfect but reasonable to be highly ranked
-        </RelevanceRating>
-        <RelevanceRating
-          v={{
-            size: 's',
-            properties: ['padding-top', 'padding-bottom'],
-          }}
-          h={{ size: 'm', properties: ['padding-left', 'padding-right'] }}
-          as="button"
-          index={3}
-          onClick={() =>
-            trackRelevanceRating({
-              id,
-              position,
-              rating: 4,
-              query,
-              page,
-              workType,
-              _queryType,
-            })
-          }
-        >
-          Completely relevant to be at this rank
-        </RelevanceRating>
-      </RelevanceRaterStyle>
-    </div>
+    showRatings && (
+      <Rating
+        ratings={ratings}
+        clickHandler={value => {
```

Comment:
I still don't understand where we use `clickHandler` vs `onClick` - from what I have seen `onClick` for custom components is pretty standard.

Reply:
This function is passed to Rating and then used with onClick

Reply:
We seem to do this differently across the code base though, and differentiating between the two seems (I think, happy to be wrong) to be unstandard.

We could use something more specific such as `onRate`?